compose-files/CentosClusterCompose.yml

```diff
@@ -12,7 +12,7 @@ services:
     networks:
       - cb-net-centos
     container_name: centos-clusterhost
-    command: --version 6.6.1 -u couchbase -p foo123! -ch clusterhost -d -os CENTOS -r -w 3
+    command: --version 6.6.1 -u couchbase -p foo123! -ch clusterhost -d -os CENTOS -r -w 3 --services data,index,query -dm 256 -im 256 -qm 256
```

Comment:
This is just a test, when actually using the script, the customer would specify it.  If i were writing a docker compose script to do this (I wouldn't, I would use the official CB images) it would be a good idea